src/panel/PoiPanel.jsx

```diff
@@ -0,0 +1,345 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Telemetry from '../libs/telemetry';
+import nconf from '@qwant/nconf-getter';
+import layouts from './layouts.js';
+
+import PoiHeader from 'src/panel/poi/PoiHeader';
+import PoiTitleImage from 'src/panel/poi/PoiTitleImage';
+import OpeningHour from 'src/components/OpeningHour';
+import OsmContribution from 'src/components/OsmContribution';
+import PoiBlockContainer from './poi_bloc/PoiBlockContainer';
+
+import Store from '../adapters/store';
+import MasqFavoriteModal from '../modals/masq_favorite_modal';
+import CategoryService from '../adapters/category_service';
+import { openShareModal } from 'src/modals/ShareModal';
+
+const store = new Store();
+const masqFavoriteModal = new MasqFavoriteModal();
+
+async function isPoiFavorite(poi) {
+  try {
+    const storePoi = await store.has(poi);
+    if (storePoi) {
+      return true;
+    }
+  } catch (e) {
+    return false;
+  }
+  return false;
+}
+
+export default class PoiPanel extends React.Component {
+  static propTypes = {
+    poi: PropTypes.object,
```

Comment:
The `isRequired` modifier would be useful, as this panel cannot exist without being given a POI.